devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -149,6 +157,7 @@ def levelEventHandler(currentLevel) {
 def updateFinalState() {
 	def level = device.currentValue("level")
 	log.debug "updateFinalState: ${level}"
+	level = level < 0 ? 0 : level > 100 ? 100 : level
```

Comment:
Please, could You explain why did You add this line ? Do Your devices report such values that need to be constrained that way?

Reply:
Yes, it's useful for some condition in compatible with Alexa and Google.

Reply:
Setting 0 and 100 will not work here. The next condition will not let You send any event in such cases.

Reply:
I have removed the code